lib/pronto/formatter/text_formatter.rb

```diff
@@ -14,9 +30,30 @@ def location(message)
         line = message.line
         lineno = line.new_lineno if line
         path = message.path
-        commit_sha = message.commit_sha[0..6] if message.commit_sha
+        commit_sha = message.commit_sha
+
+        if path || lineno
+          format_location_path(path, lineno)
+        elsif commit_sha
+          format_location_commit(commit_sha)
+        else
+          raise 'Invalid message location'
```

Comment:
No exceptions were thrown previously, it would have just returned `nil` for location if the `commit_sha` wasn't present.

Is an exception something that should be introduced here?

I wouldn't really expect a "text formatter" to throw, I would expect it to do as much as possible and degrade gracefully.

If an exception should be thrown, is this the correct way to do it? It doesn't look like theres any exception handling code around the [call to format](https://github.com/mmozuras/pronto/blob/6961ced965b522fcee87f85ba0bd075fb2c4cde6/lib/pronto.rb#L51). That would indicate that the whole app is going to crash with an unhandled exception which isn't good. Maybe the `#format` method should catch exceptions and replace them with something like `"Unable to format message. Message: #{message} Error: #{e.inspect}"`.